crates/ra_syntax/src/parser_impl/event.rs

```diff
@@ -235,6 +248,7 @@ fn n_attached_trivias<'a>(
     kind: SyntaxKind,
     trivias: impl Iterator<Item = (SyntaxKind, &'a str)>,
 ) -> usize {
+    // FIXME: parse attached trivias of CONST_DEF/TYPE_DEF
```

Comment:
@matklad it seems doc_comments for `const/type` not covered here?
```rust
/// a const
pub const A: i32 = 1;

/// a type
pub type B = i32;
```

Reply:
weird... I think we can just ad those types to this match perhaps?